vra/data_source_zone.go

```diff
@@ -46,13 +46,13 @@ func dataSourceZone() *schema.Resource {
 			},
 			"owner": {
 				Type:     schema.TypeString,
-				Optional: true,
+				Computed: true,
```

Comment:
Why we need to change it from optional to computed?

Reply:
In data source, all values are computed except the id, name and filter properties as they are read only.